Tasks/AzureCLIV1/package.json

```diff
@@ -17,8 +17,8 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "azure-pipelines-task-lib": "^3.0.6-preview.0",
-    "@types/node": "^10.17.0",
+    "azure-pipelines-task-lib": "^4.0.1-preview",
```

Comment:
please update azure-pipeline-task-lib to latest version(4.1.0) according to [guide](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/migrateNode16.md#upgrading-tasks-to-node-16)

Reply:
@max-zaytsev , are we safe with version 4.1.0?

Reply:
@ozanovus latest version of task-lib has [the same maxBuffer](https://github.com/microsoft/azure-pipelines-task-lib/blob/master/node/toolrunner.ts#L597) value as 4.0.1-preview

Reply:
thanks, updated.